superset-frontend/src/SqlLab/components/QueryTable.jsx

```diff
@@ -47,6 +47,18 @@ const defaultProps = {
   onDbClicked: () => {},
 };
 
+const StyledButtonContainer = styled.div`
+  width: 100px;
+`;
+
+const StyledLinksContainer = styled.div`
+  width: 75px;
+`;
+
+const StyledProgressBar = styled(ProgressBar)`
+  width: 75px;
```

Comment:
While I'm confident from the code that what you did here will look perfect, have you had a chance to verify this visually? The reason I ask is not fear of a regression, but instead, I'm wondering if we even _need_ this styling at all, or if the ProgressBar looks goofy without it. 

For all three of these specified pixel widths, I'm not sure from looking at the code what they're based on, or if they're really needed at all.

Reply:
Yes, there is a small change regarding UI - but imo not that significant, so given your comment below, it's worth removing these styles. Just in case, there is screenshot:
with styles:
![image](https://user-images.githubusercontent.com/47450693/100081367-3748b680-2e47-11eb-9233-a2a01a632c9a.png)

without styles:
![image](https://user-images.githubusercontent.com/47450693/100081352-344dc600-2e47-11eb-9126-6c1c200f3164.png)

Reply:
Yay, less code! I think this looks OK, and we can revisit the progress bar in particular when it's migrated from React Bootstrap to AntD soon.